config.py

```diff
@@ -21,12 +21,16 @@
 SECURITY_POST_REGISTER_VIEW = "/admin/"
 
 # Flask-Security features
-SECURITY_CONFIRMABLE = True
-SECURITY_REGISTERABLE = True
+SECURITY_CONFIRMABLE = False
+SECURITY_REGISTERABLE = False
 SECURITY_RECOVERABLE = True
 SECURITY_CHANGEABLE = True
 SQLALCHEMY_TRACK_MODIFICATIONS = False
 
 # Flask-Mail config
 MAIL_SENDGRID_API_KEY = "XXXX"
 SECURITY_EMAIL_SENDER = "noreply@retail.zap.me"
+
+SETTLEMENT_ADDRESS = ""
```

Comment:
why are these added here?

Reply:
because i added it when i was running on my testvm